GIAP.Server/Services/IdentityProviderService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Text.Json;
+using GIAP.Server.Models;
+
+namespace GIAP.Server.Services;
+
+/// Class is used as a singleton, see Program.cs
```

Comment:
Comments moeten in dit formaat, zou automatisch moeten gaan in visual studio (code) https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags